sdk/graphql-transport/test/compatability.test.ts

```diff
@@ -350,16 +350,18 @@ describe('GraphQL SuiClient compatibility', () => {
 		expect(graphQLObjects).toEqual(rpcObjects);
 	});
 
-	test.skip('queryTransactionBlocks', async () => {
+	test('queryTransactionBlocks', async () => {
 		const { nextCursor: _, ...rpcTransactions } = await toolbox.client.queryTransactionBlocks({
 			filter: {
 				FromAddress: toolbox.address(),
 			},
 			options: {
 				showBalanceChanges: true,
 				showEffects: true,
+				showRawEffects: true,
 				showEvents: true,
-				showInput: true,
+				// TODO inputs missing valueType
```

Comment:
How big of a deal is this?

Reply:
This isn't a new issue in the graphql-transport.  Basically JSON rpc returns pure inputs like `{ type: 'pure', valueType: 'u64', value: 1 }` we currently return something like `{ type: 'pure', value: [1, 0, 0, 0, 0] }`.  I'm not sure how big a deal it is, but this is a known issue